lib/renderer/init.js

```diff
@@ -121,7 +121,7 @@ if (preloadScript) {
   try {
     require(preloadScript)
   } catch (error) {
-    if (error.code === 'MODULE_NOT_FOUND') {
+    if (error.code === 'MODULE_NOT_FOUND' && error.message.indexOf(preloadScript) > -1) {
```

Comment:
I think it's a good idea, just when the preload script is loaded and some dependency from it isn't the error message will be ''Unable to load preload script: preload-script.js" which is not the real module but the stack trace will be there to help. It'll be better than the current one.